ninja/pagination.py

```diff
@@ -176,9 +176,13 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
     def add_api_operation(
         self, path: str, methods: List[str], view_func: Callable, **kwargs: Any
     ) -> None:
```

Comment:
I'm not sure about this one - only GET method
Technically you can return paginated response on any method GET/POST/etc...

Reply:
I just assume people usually use pagination on GET method and thought it's more efficient not to inject other methods to pagination.
But you're right. No need to limit possibilities.
I will rollback that thing.